packages/pyright-internal/src/backgroundThreadBase.ts

```diff
@@ -93,6 +93,13 @@ export class BackgroundThreadBase {
     }
 }
 
+export function createJsonObjectFrom(configOptions: ConfigOptions): any {
+    return {
+        ...JSON.parse(JSON.stringify(configOptions)),
+        defineConstant: configOptions.defineConstant, // This is the only field that can't be stringified.
```

Comment:
not sure why Map is not automatically serialized.

Reply:
Map is serialized when sending the data, but not when calling JSON.stringify. See this here:
https://stackoverflow.com/questions/29085197/how-do-you-json-stringify-an-es6-map

Reply:
so, issue is our `Uri` type is not serializable?